__tests__/app.test.js

```diff
@@ -235,6 +235,23 @@ describe("Comments - POST/PATCH/DELETE", () => {
                 })
             })
         })
+        test("POST: 201 - should ignore unnecessary send properties", () => {
+            const comment = {
+                body: "Is this the same Mitch that runs the seminars?",
+                author: "rogersop",
+                uselessKey: "not important at all"
+            }
+            return request(app).post(`/api/articles/4/comments`)
+            .send(comment)
+            .expect(201)
+            .then(({body}) => {
+                return db.query(`SELECT * FROM comments ORDER BY created_at DESC;`)
```

Comment:
You probably shouldn't be calling db.query in your test file you should just test the response you get back, you will know it has been added to your db by the response having an id and a created at.